core/src/main/java/com/ritense/valtimo/repository/CamundaReportingRepository.java

```diff
@@ -54,7 +54,7 @@ public InstanceCountChart getProcessInstanceCounts(String processDefinitionKey)
         Map<String, Object> parameters = new HashMap<>();
         LocalDate defaultFromDate = LocalDate.now().minusDays(14);
         Instant instant = defaultFromDate.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant();
-        parameters.put("dateFrom", Date.from(instant));
+        parameters.put("dateFrom", java.util.Date.from(instant));
```

Comment:
Why was this done? Not sure this has any benefit.

Reply:
![Screenshot 2023-07-28 at 11 49 41](https://github.com/valtimo-platform/valtimo-backend-libraries/assets/66533148/c9c40346-ce29-4506-bbf2-8d1820133388)

Reply:
I see. I didn't think the Date used here would be `java.sql.Date`.